reference/catalog.v3.yml

```
@@ -3816,7 +3816,7 @@ paths:
         **Read-Only Fields**
         * id
 
-        **Note:** The max number of metafields allowed on each product, category, variant, or brand is fifty. For more information, see [Platform Limits](https://support.bigcommerce.com/s/article/Platform-Limits) in the Help Center.
+        **Note:** The max number of metafields allowed on each product, category, variant, or brand is 250. For more information, see [Platform Limits](https://support.bigcommerce.com/s/article/Platform-Limits) in the Help Center.
```

```suggestion
        **Note:** The max number of metafields allowed on each product, category, variant, or brand is 250 per Client ID. For more information, see [Platform Limits](https://support.bigcommerce.com/s/article/Platform-Limits) in the Help Center.
```

It's actually 250 per object per client, not just 250 per object overall. Can we update this across all descriptions?

---

sure, thank you